### PR TITLE
integrate the broker adapter into the wallet data

### PR DESCRIPTION
**Pull request by rubenmodamioin2**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v2.1.0](https://github.com/in2workspace/wallet-data/releases/tag/v2.1.0) - unreleased
```
Comment (oriolcanadesin2) [inline CHANGELOG.md:7]: Add to v2.0.0

File: README.md [hunk 2]
```diff
@@ -39,29 +39,28 @@ Built using Java 17 and Spring WebFlux, the service operates on a reactive progr
...
+    WALLET-CRYPTO_DOMAIN: "http://wallet-crypto:8080"
+    WALLET-CRYPTO_URL: "/api/v2/secrets"
   ports:
     - "8086:8080"
-  links:
-    - broker-adapter
 ```
 ## Project Status
-The project is currently at version **2.0.1** and is in a stable state.
+The project is currently at version **2.1.0** and is in a stable state.
```
Comment (oriolcanadesin2) [inline README.md:63]: change to v2.0.0

File: build.gradle [hunk 3]
```diff
@@ -9,7 +9,7 @@ plugins {
 }
 
 group = 'es.in2'
-version = '2.0.0'
+version = '2.1.0'
```
Comment (oriolcanadesin2) [inline build.gradle:12]: change to v2.0.0

File: src/main/java/es/in2/walletdata/controller/UserController.java [hunk 4]
```diff
@@ -51,7 +51,7 @@ public Mono<Void> registerUser(@RequestBody @Valid UserRequest appUserRequest) {
     public Mono<UserAttribute> getUserDataByUserId(@RequestHeader(HttpHeaders.AUTHORIZATION) String authorizationHeader) {
         log.debug("UserController.getUserDataByUserId");
         log.debug("VerifiableCredentialController.getSelectableVCs()");
-        return ApplicationUtils.getUserIdFromToken(authorizationHeader)
+        return Utils.getUserIdFromToken(authorizationHeader)
```
Comment (oriolcanadesin2) [inline src/main/java/es/in2/walletdata/controller/UserController.java:54]: Why you need to use 'Utils'? Isn't a public static method?

**Review by oriolcanadesin2 — CHANGES_REQUESTED** [review]

File: src/main/java/es/in2/walletdata/controller/UserController.java (hunk 4 above)
Comment (rubenmodamioin2) [reply src/main/java/es/in2/walletdata/controller/UserController.java]: solved

File: build.gradle (hunk 3 above)
Comment (rubenmodamioin2) [reply build.gradle]: solved

File: README.md (hunk 2 above)
Comment (rubenmodamioin2) [reply README.md]: solved

File: CHANGELOG.md (hunk 1 above)
Comment (rubenmodamioin2) [reply CHANGELOG.md]: solved

**Review by oriolcanadesin2 — APPROVED** [review]

**Review by oriolcanadesin2 — APPROVED** [review]

**Pull request merged**